programmers/K번째수.js

```diff
@@ -0,0 +1,7 @@
+function solution(array, commands) {
+  let answer = commands.map((val) => {
+    let [i, j, k] = val;
+    return array.slice(i - 1, j).sort((a, b) => { return a - b; })[k - 1];
```

Comment:
콜백함수를 한줄로 코딩할 때는 중괄호와 ```return```구문이 없어도 된답니다~
```suggestion
    return array.slice(i - 1, j).sort((a, b) => a - b)[k - 1];
```